### DISCUSSION TITLE
Miboxer garden light FUTCO3Z

### DISCUSSION THREAD

**Discussion by steve41200** [Koenkk/zigbee2mqtt#17243] [Devices]
HI, is  Miboxer garden light FUTCO3Z work ?
I see one ref mi boxer who seems given the same possibility on the device page compatible but it isn't exactly the same reference
https://www.zigbee2mqtt.io/supported-devices/

I only found it on aliexpress, and if it is not compatible I cannot return them ...
And it seems the best solution for low-cost "Phillips hue type"

Thank you

**Comment by pxStrato** [comment]
Hey, I found the same lights and was wondering if someone already gave them a try and can let us know if they work with z2m?

**Comment by steve41200** [comment]
It works :)
I’ve buy 2 who works like à charme
